pennylane/templates/embeddings/amplitude.py

```diff
@@ -107,6 +107,8 @@ def circuit(f=None):
 
     """
 
+    # pylint: disable=too-few-public-methods
+
```

Comment:
I don't know why I was getting codefactor errors about this file, so I just put this here, but it's unrelated to this PR.

Reply:
Now there are 40 other similar codefactor errors, so I guess this wasn't particularly useful 🫠